src/release/breaking-changes/2-10-deprecations.md

```diff
@@ -0,0 +1,627 @@
+[`TextField`]: {{site.api}}/flutter/material/TextField-class.html
+[`TextFormField`]: {{site.api}}/flutter/material/TextFormField-class.html
+[`CupertinoTextField`]: {{site.api}}/flutter/cupertino/CupertinoTextField-class.html
+
+---
+
+### `VelocityTracker` constructor
+
+Supported by Flutter Fix: yes
+
+The default constructor for `VelocityTracker`was deprecated in v1.22.
+
+The `VelocityTracker.withKind()` should be used instead. This allows for a
+`PointerDeviceKind` to be specified for the tracker. The previous default for
+`VelocityTracker.kind` was `PointerDeviceKind.touch`.
+
+**Migration guide**
+
+Code before migration:
+<!-- skip -->
+```dart
+final VelocityTracker tracker = VelocityTracker();
+```
+
+Code after migration:
+<!-- skip -->
+```dart
+final VelocityTracker tracker = VelocityTracker.withKind(PointerDeviceKind.touch);
+```
+
+**References**
+
+API documentation:
+* [`VelocityTracker`][]
+* [`PointerDeviceKind`][]
+
+Relevant PRs:
+* Deprecated in [#66043]({{site.repo.flutter}}/pull/66043)
+* Removed in [#98541]({{site.repo.flutter}}/pull/98541)
+
+[`VelocityTracker`]: {{site.api}}/flutter/gestures/VelocityTracker-class.html
+[`PointerDeviceKind`]: {{site.api}}/flutter/dart-ui/PointerDeviceKind.html
+
+---
+
+### `DayPicker` & `MonthPicker`
+
+Supported by Flutter Fix: no
+
+The `DayPicker` and `MonthPicker` widgets were first deprecated in v1.15, and
+then extended in v1.26.
+
+They have been replaced by one comprehensive widget, `CalendarDatePicker`.
+
+These widgets were displayed using the `showDatePicker` method. This method was
+migrated to present the new `CalendarDatePicker` before this release, and so
+their final removal should not necessitate further action.
+
+**References**
+
+Design document:
+* [Material Date Picker Redesign][]
+
+API documentation:
+* [`CalendarDatePicker`][]
+* [`showDatePicker`][]
+
+Relevant issues:
+* [Issue 50133]({{site.repo.flutter}}/issues/50133)
+
+Relevant PRs:
+* Deprecated in [#50546]({{site.repo.flutter}}/issues/50546)
+* Removed in [#98543]({{site.repo.flutter}}/issues/98543)
+
+[Material Date Picker Redesign]: {{site.url}}/go/material-date-picker-redesign
+[`CalendarDatePicker`]: {{site.api}}/flutter/material/CalendarDatePicker-class.html
+[`showDatePicker`]: {{site.api}}/flutter/material/showDatePicker.html
+
+---
+
+### `FlatButton`, `RaisedButton`, & `OutlineButton`
+
+Supported by Flutter Fix: no
+
+The `FlatButton`, `RaisedButton`, and `OutlineButton` widgets were first
+deprecated in v1.20, and then extended in v1.26.
+
+They are replaced by new buttons, `TextButton`, `ElevatedButton`, and
+`OutlinedButton`. These new widgets also use new associated themes, rather than
+the generic `ButtonTheme`.
+
+<div class="table-wrapper" markdown="1">
+
+| Old Widget   | Old Theme     | New Widget      | New Theme |
+| -------------|---------------|-----------------|----------|
```

Comment:
nit: line up |'s for the OCD amongst us plz =)

Reply:
Happily!